kokoro/linux/cmake_distcheck/build.sh

```diff
@@ -49,18 +49,4 @@ fi
 
 tar -C ${DIST_WORK_ROOT} --strip-components=1 -axf ${DIST_ARCHIVE}
 
-#
-# Run tests using extracted sources
-#
-if SOURCE_DIR=${DIST_WORK_ROOT} \
-  CMAKE_GENERATOR=Ninja \
-  CTEST_PARALLEL_LEVEL=$(nproc) \
-  kokoro/common/cmake.sh; then
```

Comment:
I think we still want to do all of this, we just expect cmake.sh to return 0 I think?

Reply:
Done...I think. Based on the line for check_missing_dist_files.sh, it looks like we may not need to explicitly check the status for cmake.sh right?

Reply:
Hmm idk it makes me nervous to assume that (it could just as easily be that check_missing_dist_files is setup to silently fail).  Can you try intentionally breaking the cmake build and see if this test breaks?

Reply:
I think you alluded to this, but I just want to clarify: we don't need to explicitly check the return value of `cmake.sh`, if it fails the error will propagate up and cause a test failure?

Reply:
Yeah, the standard `set -e` handles this (line 5).